src/openrct2/core/Path.cpp

```diff
@@ -202,42 +191,15 @@ namespace Path
         return lastDot;
     }
 
-    utf8* GetAbsolute(utf8* buffer, size_t bufferSize, const utf8* relativePath)
+    utf8* GetAbsolutePath(utf8* buffer, size_t bufferSize, const utf8* relativePath)
     {
```

Comment:
I don't agree with this rename. We are already in the path namespace. Also you've only renamed one of the overloads

Reply:
I thought I had dropped this renaming and left the new name only on Platform, but seems like I hadn't. Just rewrote it.